torch/_inductor/fx_passes/quantization.py

```diff
@@ -136,6 +139,9 @@ def get_qlinear(users=1):
         KeywordArg("postop_algorithm"),
         _users=users,
     )
+    return _may_generate_pattern_with_dtype_convert(
+        computation_call, dtype=KeywordArg("to_float"), dtype_convert=is_bf16, users=2
```

Comment:
I don't think we should add this in `get_qlinear`, especially the `users` are hardcoded as 2. The Linear-GeLU int8-mixed-bf16 has `to_fp32` after qlinear is because of the post op gelu, can we move it to the pattern of post op GeLU?

Reply:
I think for `none` and `tanh`, the users should be different? similar as the difference we see in the int8-mixed-fp32?

Reply:
Thanks for the comment! `none` and `tanh` has different `users`. I've made a change accordingly.